oauth2_http/java/com/google/auth/oauth2/TokenVerifier.java

```diff
@@ -314,17 +314,13 @@ public static class JsonWebKey {
     public Map<String, PublicKey> load(String certificateUrl) throws Exception {
       HttpTransport httpTransport = httpTransportFactory.create();
       JsonWebKeySet jwks;
-      try {
-        HttpRequest request =
-            httpTransport
-                .createRequestFactory()
-                .buildGetRequest(new GenericUrl(certificateUrl))
-                .setParser(OAuth2Utils.JSON_FACTORY.createJsonObjectParser());
-        HttpResponse response = request.execute();
-        jwks = response.parseAs(JsonWebKeySet.class);
-      } catch (IOException io) {
-        return ImmutableMap.of();
-      }
+      HttpRequest request =
```

Comment:
Do try catch like before but throw a good exception instead of swallowing?

Reply:
we don't change it and throw the same thing - so there is no need. 

In the old library, in comparison, we have a logger and at least can trace before rethrowing